ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml

```
@@ -1497,6 +1522,29 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: ipi-gcp
+- as: amd-notebooks-e2e-tests
+  run_if_changed: (base\/ubi9-python-3.9\/*)|(jupyter\/minimal\/ubi9-python-3.9\/*)|(jupyter\/datascience\/ubi9-python-3.9\/*)|(amd\/*)
+  steps:
+    cluster_profile: gcp-opendatahub
+    test:
+    - as: amd-jupyter-ubi9-python-3.9-test-e2e
+      cli: latest
+      commands: |
+        oc new-project amd-jupyter-ubi9-python-3-9
+        IFS=':' read -r -a NOTEBOOK_IMAGE <<< "${JUPYTER_AMD_IMAGE}"
+        make deploy8-amd-ubi9-python-3.9 \
```

```suggestion
        make deploy9-amd-ubi9-python-3.9 \
```
Not tested, but to be sure please check it localy with the makefile